src/utils/wallet.js

```diff
@@ -30,13 +32,60 @@ const KEY_WALLET_ACCOUNTS = KEY_UNIQUE_PREFIX + 'wallet:accounts_v2'
 const KEY_ACTIVE_ACCOUNT_ID = KEY_UNIQUE_PREFIX + 'wallet:active_account_id_v2'
 const ACCESS_KEY_FUNDING_AMOUNT = process.env.REACT_APP_ACCESS_KEY_FUNDING_AMOUNT || '100000000'
 const ACCOUNT_ID_REGEX = /^(([a-z\d]+[-_])*[a-z\d]+\.)*([a-z\d]+[-_])*[a-z\d]+$/
+const ACCOUNT_NO_CODE_HASH = '11111111111111111111111111111111'
+const METHOD_NAMES_LAK = ["add_request", "add_request_and_confirm", "delete_request", "confirm"]
+const METHOD_NAMES_CONFIRM = ["confirm"]
+
 export const keyAccountConfirmed = (accountId) => `wallet.account:${accountId}:${NETWORK_ID}:confirmed`
 
 const WALLET_METADATA_METHOD = '__wallet__metadata'
 
 export const ACCOUNT_CHECK_TIMEOUT = 500
 export const TRANSACTIONS_REFRESH_INTERVAL = 10000
 
+/********************************
+Linkdrop data
+********************************/
+export const getLinkdropData = () => {
+    return JSON.parse(localStorage.getItem(`__linkdropData`) || `{}`)
+}
+export const setLinkdropData = (data) => {
+    return localStorage.setItem(`__linkdropData`, JSON.stringify(data))
+}
+/********************************
+Managing 2fa requests
```

Comment:
wallet.js is already pretty big as is. Seems like best to move 2FA stuff into separate module, e.g. `utils/2fa.js`.

In general most cases when it seems like section comment is needed it makes more sense to just move involved code into separate module, component, class, function, etc.

Reply:
in progress